src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyles = sourceString
+    .split(';')
+    .filter((declaration) => declaration.trim())
+    .map((declaration) => declaration.split(':'))
+    .reduce((accumulator, [key, value]) => {
+      accumulator[key.trim()] = value.trim();
+
+      return accumulator;
+    }, {});
```

Comment:
The chained methods should have the same indentation to improve readability. Make sure that each method in the chain starts on a new line and is indented to the same level as the others.